app/assets/main/react_components/footprint_form/ResultPage.jsx

```diff
@@ -75,9 +79,11 @@ const ResultPage = ({ result, texts, lang, page, onPageChange, currency }) => {
                 onAnswerGiven={onPageChange}
                 stylingClasses={"w-2/3 " + (page === 2 && "button-cta")}
             />
-            {page === 3 && 
+            { page === 3 && 
                 <div className={"inject-link pt-4"}
-                    dangerouslySetInnerHTML={{__html: sanitizeHtml(texts.registrationsText.accept_policies)}}>
+                    dangerouslySetInnerHTML={{
+                        __html: sanitizeHtml(texts.registrationsText.accept_policies.replace("<a>","<a href='"+ ((slug==='en' || slug===null) ? "" : "/" + slug) +"/privacy-policy' target='_blank'>" ))}}
```

Comment:
```suggestion
                        __html: sanitizeHtml(texts.registrationsText.accept_policies.replace("<a>","<a href='"+ ((slug === 'en' || slug === null) ? "" : "/" + slug) +"/privacy-policy' target='_blank'>" ))}}
```

Tänker approvea men du kan ändra om du vill ¯\_(ツ)_/¯